packages/widget/src/controllers/wallet-manager/manager.ts

```diff
@@ -61,6 +61,9 @@ export class WalletController implements ReactiveController {
   disconnectWallet = (): void => {
     this.disconnectEvmWallet();
     this.disconnectSubstrateWallet();
+    this.host.dispatchEvent(
```

Comment:
Don't think you need this. TransferController is subscribed to walletContext changes https://github.com/sygmaprotocol/sygma-widget/blob/dev/packages/widget/src/controllers/transfers/fungible-token-transfer.ts#L71

so you can check is wallet is disconnected on every change